app/__init__.py

```diff
@@ -1,7 +1,36 @@
+
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os
+
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv()
 
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
+    if not test_config:
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
```

Comment:
Since we need to set `'SQLALCHEMY_TRACK_MODIFICATIONS'` to false in the if and the else block, we can bring this line outside of the conditional statements so there's no repetition.